src/scss/_global.scss

```diff
@@ -34,7 +36,14 @@ $paragraph-line-height-tablet: 1.5rem;
 $paragraph-line-height-desktop: 1.5rem;
 
 // Structure Variables
-$content-margin: 1rem;
+$content-margin: 2rem;
+
+// Table Styles
+
+tbody {
+  font-family: monospace;
```

Comment:
We may want to play around with different monospace fonts. I know this is something I messed around with in a previous version of the WordPress theme.

I also tried using the regular Lato font with `font-variant: tabular-nums;`, but I'm not sure if it really did anything. Want to play with it and see what you think looks best?